db/migrations/0001_initial.py

```diff
@@ -41,4 +59,55 @@
                 ("name", models.CharField(max_length=255, unique=True)),
             ],
         ),
+        migrations.CreateModel(
+            name="Movie",
+            fields=[
+                (
+                    "id",
+                    models.AutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("title", models.CharField(max_length=255)),
+                ("description", models.TextField()),
+                (
+                    "actors",
+                    models.ManyToManyField(related_name="movies", to="db.Actor"),
+                ),
+                (
+                    "genres",
+                    models.ManyToManyField(related_name="movies", to="db.Genre"),
+                ),
+            ],
+        ),
+        migrations.CreateModel(
+            name="MovieSession",
+            fields=[
+                (
+                    "id",
+                    models.AutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("show_time", models.DateTimeField()),
+                (
+                    "cinema_hall",
+                    models.ForeignKey(
+                        on_delete=django.db.models.deletion.CASCADE, to="db.cinemahall"
```

Comment:
The `to` parameter in the `ForeignKey` should reference the model name in lowercase, as Django automatically converts model names to lowercase for the database table names. Change `to="db.cinemahall"` to `to="db.cinemahall"`.